deepchem/trans/transformers.py

```diff
@@ -2469,3 +2469,62 @@ def median_filter(self, size):
     image = Image.fromarray(self.Image)
     image = image.filter(ImageFilter.MedianFilter(size=size))
     return np.array(image)
+
+
```

Comment:
Make sure to add the new transformer to the transformer docs and to add some unit tests

Reply:
Will do!